### ISSUE TITLE
Option to remove motion on stacking tearsheets (narrow opening on wide)

### ISSUE THREAD

**Issue by Lily-O-ibm** [carbon-design-system/ibm-products#4066]
**Goal:** to have the ability for our Turbonomic devs to remove motion on stacking tearsheets. We are utilizing a stacked narrow tearsheet opening on a wide tearsheet. This is currently creating a strange motion zoom effect, perhaps a bug or maybe due to the varying sizes in tearsheets we are using? Also, in our case, we are opening two differently sized tearsheets (narrow on wide), and this also causes the bottom of the UI to be visible, and the bottom tearsheet is not staying anchored in place.

**We need:** To have the option to remove the built in default motion and fix this issue along with having bottom of tearsheet anchored, I consulted with Matt Gallo, and have been told to open 2 issues, one in https://github.com/carbon-design-system/ibm-products/ (main carbon/PAL repo for fix) and one internal PAL repo for guidance update https://github.ibm.com/CDAI-design/pal - and to tag Matthew Gallo and Elysia Hwang, PAL UX engineers.

Once fixed, I will bring back to our Turbo dev team, and ask them to update.

tagging @matthewgallo  and @elycheea as spoke to Matt G. about this. Please let me know if more context or info is needed.

**Issue as is in screenshot and .gif file:**
![layers in order](https://github.com/carbon-design-system/ibm-products/assets/156940847/e0c8cda8-0b89-403e-99fc-2238f361cc69)

![motion in stacking tearsheets issue](https://github.com/carbon-design-system/ibm-products/assets/156940847/b7394ee5-5b12-408c-8a53-a77b3c565c02)

## Additional information

Please let me know if you need to view in our Turbo environment - we can meet and I can share info. 

In this case, the wide tearsheet animation and position should not occur at all. May need to look at depth, position, and size from TearsheetShell to achieve this.

```[tasklist]
### Tasks
- [ ] If stacking a narrow tearsheet on a wide tearsheet, wide tearsheet should not visibly change
- [ ] Verify that this works for both adjacent tearsheets and nested tearsheets
```

**Comment by carrenelloyd** [comment]
Assuming we will investigate why the stacked narrow tearsheet has a strange motion zoom effect before removing it entirely.

**Comment by elycheea** [comment]
@carrenelloyd It does the zoom because the it’s still a Tearsheet so the behavior is for them to animate with a stack.

**Comment by elycheea** [comment]
@Lily-O-ibm I can work with you to define how we should address this. Our team looked at this and agrees with your suggestions here but want to make sure how we’re able to understand the different scenarios we would need to accommodate before we implement a fix for just this situation.

**Comment by Lily-O-ibm** [comment]
thanks @elycheea will reach out, guessing a sync to demo our situation may be helpful?

**Comment by matthewgallo** [comment]
This is complete now and available in `2.31.0-alpha.5`.
